src/main/java/seedu/duke/entities/Dish.java

```diff
@@ -2,7 +2,7 @@
 
 public class Dish {
     private String name;
-    private double price;
+    public double price;
```

Comment:
Instead of making price public I think it is better to use a getter for price instead

Reply:
yes

Reply:
Dish should have a getter already?